jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/processinstance/ProcessInstanceInfo.java

```diff
@@ -103,6 +104,16 @@ public ProcessInstanceInfo(ProcessInstance processInstance) {
         this.processInstance = processInstance;
         this.processId = processInstance.getProcessId();
         startDate = new Date();
```

Comment:
You can move this line inside the first condition, for clarity and not overwriting when getStartDate is not null